paper_eval.py

```diff
@@ -89,7 +102,21 @@
 
 logger.info("Setting up dataloaders")
 
-df = pd.read_parquet(data_dir / parquet_file)
+
+logger.info("Reading dataset")
+files = sorted(glob(f"{parquet_file}/**/*.parquet"))[:10]
+df_list = []
+for f in tqdm(files):
+    _data = pd.read_parquet(f, engine="fastparquet")
+    _data_pivot = process_parquet(_data)
+    _data_pivot.reset_index(inplace=True)
+    df_list.append(_data_pivot)
+    del _data, _data_pivot
```

Comment:
does this truly have impact? In the next iteration this variable anyway gets overwritten?

Reply:
fixed ff255099e5ead303211fac3442cb6fef3f2e49c5